transport-native-epoll/src/main/c/netty_unix_socket.c

```diff
@@ -708,6 +730,10 @@ static JNINativeMethod* createDynamicMethodsTable(const char* packagePrefix) {
     dynamicMethod->signature = netty_unix_util_prepend("(IJII)L", dynamicTypeName);
     dynamicMethod->fnPtr = (void *) netty_unix_socket_recvFromAddress;
     free(dynamicTypeName);
+    ++dynamicMethod;
+    dynamicMethod->name = "peerCredentials";
```

Comment:
construct the method name using the `netty_unix_util_prepend` like the other methods signatures. This allows us to add a prefix to the package name and is to support shading.